bin/cloneDataDump.js

```diff
@@ -0,0 +1,46 @@
+const clone = require('git-clone');
+const rimraf = require('rimraf');
+const fs = require('fs');
+const path = require('path');
+
+const tmpDestination = path.resolve(`.${path.sep}tmp${path.sep}dataDump${path.sep}`);
+const tmpData = path.resolve(`${tmpDestination}${path.sep}data${path.sep}`);
+const employersDestination = path.resolve(`.${path.sep}src${path.sep}data${path.sep}employers${path.sep}`);
+
+const cleanupTmpDirectory = () => rimraf(tmpDestination, () => {
+  // eslint-disable-next-line no-console
+  console.log(`${tmpDestination} has been removed.\n`);
+});
+
+const transferEmployerFiles = () => {
+  fs.readdirSync(tmpData).forEach((file) => {
+    if (path.extname(file) === '.json') {
+      fs.renameSync(`${tmpData}${path.sep}${file}`, `${employersDestination}${path.sep}${file}`, (err) => {
+        if (err) throw err;
+      });
+    }
+  });
+
+  // eslint-disable-next-line no-console
+  console.log(`dataDump employers information moved to ${employersDestination}\n`);
+};
+
+const cloneRepository = (repository) => clone(repository, tmpDestination, { shallow: true },
+  () => {
+    // eslint-disable-next-line no-console
+    console.log(`${repository} cloned to ${tmpDestination}\n`);
+    fs.accessSync(tmpData, fs.constants.F_OK, (err) => {
```

Comment:
What's happening with this? It's not immediately clear until I read the console statement. A way to make it more expressive could be to assign it a function name, like `const checkForEmployerDirectory = () => fs.accessSync(.....`. It may also make sense to fire this inside of `transferEmployerFiles()`, but I don't feel as strongly about that part.

Reply:
I think this is half working. The script is supposed to double check that things were copied, but it looks like it only works if there's actually an error.

I dig on moving it though.